app/javascript/components/components/Footer.js

```diff
@@ -1,9 +1,46 @@
 import React, { Component } from 'react'
+import { Nav, 
+  NavItem, 
+  Navbar, 
+  NavbarBrand, 
+  NavbarToggler, 
+  Collapse, 
+  NavLink, 
+  } from 'reactstrap';
 
 export default class Footer extends Component {
   render() {
     return (
-      <h3>This is a Footer</h3>
+      <footer>
+        <Navbar
+          color="secondary"
+          expand="md"
+          fixed=""
+          light
+        >
+          <NavbarBrand href="/">
+            Wayne Enterprise
+          </NavbarBrand>
+          <NavbarToggler onClick={function noRefCheck(){}} />
+          <Collapse navbar>
+            <Nav
+              className="me-auto"
+              navbar
+            >
+              <NavItem>
+                <NavLink href="/">
```

Comment:
NavLinks with internal navigation (other react components) should use the react-router-dom. 
`Navlink to="/"`